src/PlayerManager.cpp

```diff
@@ -588,10 +590,16 @@ void PlayerManager::enterPlayerVehicle(CNSocket* sock, CNPacketData* data) {
     INITSTRUCT(sP_FE2CL_PC_VEHICLE_ON_SUCC, response);
     PlayerView plrv = PlayerManager::players[sock];
 
-    // send to other players
-    //for (CNSocket* otherSock : plrv.viewable) {
-    //    otherSock->sendPacket((void*)&response, P_FE2CL_PC_VEHICLE_ON_SUCC, sizeof(sP_FE2CL_PC_VEHICLE_ON_SUCC));
-    //}
+    //send to other players
+    INITSTRUCT(sP_FE2CL_PC_EQUIP_CHANGE, pkt);
+    pkt.EquipSlotItem.iType = 1;
+    pkt.iEquipSlotNum = 8;
+    for (CNSocket* otherSock : plrv.viewable) {
+        otherSock->sendPacket((void*)&pkt, P_FE2CL_PC_VEHICLE_ON_SUCC, sizeof(sP_FE2CL_PC_EQUIP_CHANGE));
```

Comment:
watch out: P_FE2CL_PC_VEHICLE_ON_SUCC vs sP_FE2CL_PC_EQUIP_CHANGE

Reply:
Fixed in my latest commit.